README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Shawn's Brief Biography:
+- **Major**: *Philosophy* & *Political Science*
+- **Year**: *Senior*
+- **Currently located**: *New Jersey*
+- Have you ever wondered if the Divorce rate in Maine correlates with the Per capita consumption of margarine? Fear not, check out this website on: [spurious relationships](https://www.tylervigen.com/spurious-correlations)
+
+### GitHub Reflections:
+- **To start**, I was completely lost with the setup until I watched the workflow video, which allowed me to get to the 'My name is Shawn Kim' step of the HW (as well as uploading the homework).
+- **Then**, I had trouble initializing Markdown and subsequently watched some YouTube videos to help but to no avail. A mistake I made was changing the README.md file into an Rmd file. After working and testing with Rmd, I realized that this it is not necessary.
+- **Next**, I utilized the helpful [Markdown guide](https://guides.github.com/features/mastering-markdown/) to include four examples.
+- **The most difficult step** was adding an image. I tried many methods I could find online (YouTube, stack overflow, etc.) but none seemed to work. I was messing around with myrepo to the point where I could not even upload the simple 'My name is...' step. So, I had to delete the entire repository and start from scratch.
+  - **If** anyone can offer help with embedding an image, please let me know!
```

Comment:
To reiterate what your peers have said so far, you'll need to upload your file through your RStudio and commit it to github. Then you use ![description](image.jpeg) to insert the image.